.gitignore

```
@@ -1,5 +1,5 @@
-/.idea/
-/log/
-/target/
-/*.iml
-/*.log
+.idea
+log
+target
+*.iml
+*.log
```

измените на следующий вариант
```suggestion
/.idea/
/logs/
/target/
*.iml
/*.log
```
дальше выполните в терминале Intelij: 
git rm --cached -r .
потом:
git add .

и пуште в эту ветку